Terminal.Gui/Views/TableView/TableView.cs

```diff
@@ -315,13 +315,16 @@ private int GetHeaderHeightIfAny ()
 		/// <returns></returns>
 		private int GetHeaderHeight ()
 		{
-			int heightRequired = 1;
```

Comment:
Could this be simplified by just adding the following at the top of the method instead?  Typically having less nesting is easier to read and maintain so would appreciate if you could make this change.

```csharp
if (Style.HideHeaders) {
    return 0;
}
```